src/services/communicator/index.ts

```diff
@@ -218,9 +218,22 @@ class Communicator {
         timestamp: actor.timestamp,
         connectionId: actor.connectionId,
         userId: actor.clientId,
-        color: this.realtime.getActorColor(actor.customProperties.slotIndex),
+        color: this.realtime.getSlotColor(actor.customProperties.slotIndex),
       };
     });
+    const getSlot = (id : string) => {
+      return actors[id]?.customProperties?.slotIndex;
+    };
+    const userListForSdk = this.userList.map((user: User) => {
+      const slotIndex = getSlot(user.id) ? getSlot(user.id) : 16;
```

Comment:
It's not good to use magic numbers, maybe you can use MeetingColors and return the gray directly.

```suggestion
      const slotIndex = getSlot(user.id) ?? MeetingColors['gray'];
```